src/qt/pivx/settings/settingswalletoptionswidget.cpp

```diff
@@ -79,11 +76,27 @@ void SettingsWalletOptionsWidget::setMapper(QDataWidgetMapper *mapper){
     mapper->addMapping(ui->lineEditPort, OptionsModel::ProxyPort);
 }
 
+void SettingsWalletOptionsWidget::setWalletModel(WalletModel* model)
+{
+    PWidget::setWalletModel(model);
```

Comment:
Can override `loadWalletModel` to avoid doing this parent call.

Essentially, `PWidget::setWalletModel` calls internally to `loadWalletModel` which is the function that has to be overrode by its child if any setup is needed after the walletModel set (same workflow applies to the `clientModel` set).